tests/test.mjs

```diff
@@ -102,11 +116,67 @@ async function getFullReport(page) {
 }
 
 async function startRequestProfiling(page) {
+  const regex = /(.*)\/changeset\/([\w\d]+)(\/.*)?/;
   page.on("requestfinished", async (request) => {
     const url = request.url();
     if (!requests[url]) {
       requests[url] = [];
     }
     requests[url].push(request.timing());
+    if (needChangesetId) {
+      const match = url.match(regex);
+      if (match) {
+        const changesetId = match[2];
+        console.log(`Changeset id found: ${changesetId}`);
+      }
+    }
```

Comment:
What's the point of this? It doesn't seem used?

Reply:
that's how the changeset Id would be retrieved, once found it would be logged to the console. Granted, now that I look at it I'd rather see if we can request getLatestChangesetId as part of the teardownBackend function...